databricks/koalas/window.py

```diff
@@ -1226,6 +1234,12 @@ def _apply_as_series_or_frame(self, func):
             _, name = index_map
```

Comment:
@itholic, can you reuse `RollingGroupby._apply_as_series_or_frame` here while we're here? You can do:

```
_apply_as_series_or_frame = RollingGroupby._apply_as_series_or_frame
```

Reply:
For `Expanding` too.

Reply:
@HyukjinKwon thanks for the review! i applied them :)